tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/02_importFile.js

```diff
@@ -0,0 +1,142 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+// Import utils
+const helper = require('@utils/helpers');
+const files = require('@utils/files');
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const importPage = require('@pages/BO/advancedParameters/import');
+const customersPage = require('@pages/BO/customers');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_advancedParameters_import_importFile';
+
+// Variables
+let browserContext;
+let page;
+
+// Variable Used in the download / rename / upload test
+let sampleFilePath;
+const sampleFile = {
+  args:
+    {
+      dropdownValue: 'Customers',
+      type: 'customers_import',
+    },
+};
+const newPath = `./${sampleFile.args.type}.csv`;
+
+// Variables used for assertions
+const importModalTitle = 'Importing your data...';
+const importPanelTitle = 'Match your data';
+const emailToCheck = 'Tiger.Lily@prestashop.com';
```

Comment:
I guess it's the data from fixtures, so same thing it should be in a demo data file. Also is this data always the same ?

Reply:
done